code/migration/src/main/scala/com/namely/chiefofstate/migration/versions/v1/V1.scala

```diff
@@ -141,16 +142,12 @@ object V1 {
    * @param journalJdbcConfig the database config
    */
   private[v1] def createTable(journalJdbcConfig: DatabaseConfig[JdbcProfile]): Unit = {
-    val stmt = sqlu"""
-        CREATE TABLE #$tempTable(
-          projection_name VARCHAR(255) NOT NULL,
-          projection_key VARCHAR(255) NOT NULL,
-          current_offset VARCHAR(255) NOT NULL,
-          manifest VARCHAR(4) NOT NULL,
-          mergeable BOOLEAN NOT NULL,
-          last_updated BIGINT NOT NULL,
-          PRIMARY KEY(projection_name, projection_key)
-        )""".withPinnedSession.transactionally
+    // TODO: consider a "current schema" object so v1
+    // migration isn't importing something from v2
+    val stmt = SchemasUtil
```

Comment:
I suggest we move the SchemasUtil from v2 then since we are using it everywhere now.

Reply:
I think you mean V1Schema here.